lib/modules/apostrophe-dialog-box-apos-rich-text-widgets/index.js

```diff
@@ -1,4 +1,19 @@
 module.exports = {
   improve: 'apostrophe-rich-text-widgets',
-  dialogBox: false
+  sanitizeHtml: {
```

Comment:
This is a pretty heavy handed change to the project .. what is the absolute minimum options we need to pass to `sanitizeHtml` ?